implants/lib/eldritch/src/file.rs

```diff
@@ -150,7 +150,7 @@ fn methods(builder: &mut MethodsBuilder) {
         remove_impl::remove(path)?;
         Ok(NoneType{})
     }
-    fn rename(this: FileLibrary, old: String, new: String) -> anyhow::Result<NoneType> {
+    fn moveto(this: FileLibrary, old: String, new: String) -> anyhow::Result<NoneType> {
```

Comment:
Imo `move` looks/sounds better, but up to you.

Reply:
Ope - tried to make this change.
We made it moveto since move is a restricted word in rust.
So can't have functions named move. 💀